lib/rules/cli/4.1.yaml

```diff
@@ -1301,6 +1301,7 @@
     :node_name: <value>
     :output: --output <value>
     :overwrite: --overwrite=<value>
+    :l: -l <value>
```

Comment:
Sort it in alphanumeric order.
And you need to add it to 4.4.yaml as well.

Reply:
Alphanumeric is done but 4.4,4.3 and 4.2 is all the way symlinked to 4.1 so the changes are being done in 4.1. Also the "-l" is supported on 4.1 and 4.4 as well